index.html

```diff
@@ -0,0 +1,386 @@
+
+							<p class="text-light text-small">
+								Try creating your own work using open source license rather than
+								just watching at it.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_opportunities.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 medium-paragraph-size"
+								id="program-task">
+								Ignite
+							</h3>
+
+							<p class="text-light text-small">
+								get opportunities to network with CC affiliates around the
+								world, also after the summit.
+							</p>
+						</div>
+					</div>
+					<h4 class="see-program text-center m-1 py-2 paragraph-size">
+						SEE THE WHOLE PROGRAM
+					</h4>
+					<div
+						class="shadow-sm p-2 mb-2 bg-danger mt-4 container"
+						id="join-program">
+						<h4 class="text-center text-white p-1 paragraph-size">
+							Join Victoria Alumni 2023 Summit
+						</h4>
+					</div>
+				</section>
+				<!-- feature speaker -->
+				<section class="feature-speaker-container bgColor-2 py-2">
+					<div class="speaker-content container-fluid">
+						
+							<h4 class="mt-3 text-dark text-center head-font-size">Feature Speaker</h4>
+						
+						<div class="d-flex justify-content-center"><hr class="bar bg-danger"> </hr></div>
+<!-- speakers mobile and desktop -->
+						<div class="feature-speaker-bio-mobile"></div>
+						<div class="feature-speaker-bio"> </div>
+					
+							<button class="btn btn-light text-center textColor-dark w-100 mb-4 paragraph-size" id="join-program">more <span class="text-danger"> <i class="fa fa-chevron-down fa-lg"></i> </span> </button>
+					</div>
+				</section>
+			</main>
+
+			<footer class="footer-content-hidden">
+				<div class="bgColor3 text-center text-light shadow  py-2">
+					<h3 class="head-font-size text-light text-center">Partners</h3>
+
+					<ul class="partner d-flex justify-content-between align-items-center">
+						<li class="mx-4">
+							<img
+								class="partner-img partner-image-size-1"
+								src="./asset/images/spike-gold.png"
+								alt="" />
+						</li>
+						<li class="mx-4">
+							<img
+								class="partner-img partner-image-size-2"
+								src="./asset/images/Speke-Resort-logo.png"
+								alt="" />
+						</li>
+					</ul>
+				</div>
+
+				<div
+					class="text-center shadow p-2  d-flex justify-content-around footer-summit-credit bg-dark">
+					<h6 class="text-small text-light">
+						<img
+							class="victoria-logo"
+							src="./asset/images/victoria-university-logo.png"
+							alt="Victoria-university-logo" />
+						Victoria university alumni
+					</h6>
+					<h6 class="text-small text-light">
+						2023 victoria university kampala.@copy Rights Reserved.
+					</h6>
+				</div>
+			</footer>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure that the Home page has a footer with a white background so it looks identical to the provided design